src/components/application/application.js

```diff
@@ -1,9 +1,25 @@
 import './application.scss';
 
-function Application() {
+function Application({ logout, user = {} }) {
   return (
     <div className="application">
-      Login Challenge
+      <header className="header">
+        <h1>Application</h1>
+        <div className="user">
+          <p className="userLink">{user.email}</p>
```

Comment:
what's the significance of this `userLink` class? It doesn't appear to be subclassing a tag which could be a link (it's just a `p`)?

Reply:
Well… I figured it might be a link eventually, just didn't want the compiler complaining about a link without a href.

Reply:
I updated it to userEmail 👍